superset/utils/core.py

```diff
@@ -213,6 +213,29 @@ def dttm_from_timtuple(d):
         d.tm_year, d.tm_mon, d.tm_mday, d.tm_hour, d.tm_min, d.tm_sec)
 
 
+def decode_iso_dttm(o):
```

Comment:
Mmmh, can we just decode the ISOs values under the keys we know may be ISOs? If we want to cache to be determined from the query_obj, we'd need the relative time expressions here too, haven't given too much thought to this, but we should make a conscious decision as to whether query_obj would need to know about relative time exprs.